public/styles.css

```diff
@@ -356,13 +353,18 @@ a {
 /* Main content styling */
 .main-content {
     margin-left: 210px; /* Same as the width of the sidenav */
```

Comment:
This is the parent div, under which all child divs will reside. This div is the reason why the top of the sidebar items were cutoff on a smaller screens. It is now set to always have 90% of the height of the view(is also responsive), with a 100px padding at the top and bottom to give a sense of space.